bokeh/models/plots.py

```diff
@@ -795,6 +809,58 @@ def _scale(cls, scale: Literal["auto", "linear", "log", "categorical"]) -> Scale
     suppress all of the actions except the Reset event.
     """)
 
+class GridPlot(LayoutDOM):
+    """ """
+
+    toolbar = Instance(ToolbarBase, default=lambda: Toolbar(), help="""
+    The toolbar associated with this grid plot, which holds all the tools.
+    It is automatically created with the plot if necessary.
+    """)
+
+    toolbar_location = Nullable(Enum(Location), default="above", help="""
+    Indicates where the layout the toolbar will be located. If set to None,
+    no toolbar will be attached to the grid plot.
+    """)
+
+    children = List(Either(
+        Tuple(Instance(LayoutDOM), Int, Int),
+        Tuple(Instance(LayoutDOM), Int, Int, Int, Int)), default=[], help="""
+    A list of subplots with their associated position in the grid, row and column
+    index and optional row and column spans (the defaul span is 1).
+    """)
+
+    rows = Either(QuickTrackSizing, Dict(IntOrString, RowSizing), default="auto", help="""
+    Describes how the grid should maintain its rows' heights.
+
+    .. note::
```

Comment:
`.. warning::` I think (and elsewhere)

Reply:
It's a copy from `GridBox`. As I mentioned earlier, this will need refinement and it isn't clear to me if this will be kept at all.

Reply:
Sure, but all things equal, changing it now means not having to remember it later, either way.